lib/ssl/doc/src/ssl.xml

```diff
@@ -861,6 +897,22 @@ fun(srp, Username :: string(), UserState :: term()) ->
       client certificate is requested. For more details see the <seealso marker="#client_signature_algs">corresponding client option</seealso>.
       </p> </item>
 
+      <tag><c>{signature_algs_cert, [signature_scheme()]}</c></tag>
+      <item>
+	<p>
+	  <url href="http://www.ietf.org/rfc/rfc8446.txt#section-4.2.3">TLS 1.3 (RFC 5246 Section 4.2.3)</url>provides two extensions for indicating which signature
+	  algorithms may be used in digital signatures.  The
+	  "signature_algorithms_cert" extension applies to signatures in
+	  certificates, and the "signature_algorithms" extension, which
+	  originally appeared in TLS 1.2, applies to signatures in
+	  CertificateVerify messages.
+	</p>
+	<p>
+	  The server will send a signature_algorithms_cert extension (CertificateRequest),
+	  if TLS version 1.3 or later is used.
+	</p>
```

Comment:
I think that we need to describe what the default value looks like, and I am not sure that we should send it as default.
Maybe we should have an option like {signature_algs_cert, [signature_scheme() | all_supported] } but by default it is not sent at all so that the signature_algorithm extension is used instead.  This might be good for higher interop and smoother  transition. The option all_supported  could be the behavior of the current implementation. Then we could specify the maximum included algorithms and say that they may be filtered due to the used cryptolib.